picard/ui/tablebaseddialog.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+# Copyright (C) 2020 Ray Bouchard
```

Comment:
This should have the copyrights from the original copied over, since a lot of this code got just copied as well.